arch/platform/jn516x/platform-conf.h

```diff
@@ -314,4 +314,8 @@ DR1174+DR1175:
 #define LEDS_GP2      64
 #define LEDS_GP3      128
 #define LEDS_CONF_ALL 255
+
+/* Platform-specific define for the end of the stack region */
+#define STACK_CONF_ORIGIN     ((void *)0x4008000)
```

Comment:
do these need to be hard-coded in `.h` file, aren't there ways to ask the linker and then pass the flag at compile time (CLFAGS)?

Reply:
Not sure what exactly / technically you have mind here?

Reply:
I was just wondering if there isn't a systematic way to find the end of stack from linkerscript, and use it as define here, instead of hardcoding it in the .h file. This way, if linker / linkscript changes, this doesn't break

Reply:
Technically it looks like it's possible to do this in the linker script like this.
`_stack_origin = ORIGIN(RAM) + LENGTH(RAM);`

Thanks. I will do it for the platforms that I can (not for msp430). For JN516x I can do it in Makefile - there is a symbol called `_ram_top` of which I'll make an alias.

Reply:
sounds great!